examples/correlation_alignment_example.py

```diff
@@ -64,15 +64,15 @@
         fig = plt.figure(figsize=(11, 7))
         gs = plt.GridSpec(1, 4, figure=fig, wspace=0.4)
         # }}}
-        fig.suptitle(fl.basename)
+        fig.suptitle("Data Processing " + fl.basename)
         fl.next("Data Processing", fig=fig)
         data = psd.fake_data(
             expression, OrderedDict(orderedDict), signal_pathway
         )
         data.reorder([indirect, "t2"], first=False)
         data.ft("t2")
         data /= np.sqrt(psd.ndshape(data)["t2"]) * data.get_ft_prop("t2", "dt")
-        fl.DCCT(data, bbox=gs[0], title="Raw Data")
+        psd.DCCT(data, bbox=gs[0], fig=fig, title="Raw Data")
```

Comment:
use updated DCCT format

Reply:
why is this better?

Reply:
using fl doesn't seem to properly title the dcct (instead it uses the title used in the fl.next)